src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -108,13 +108,15 @@ function MoneyRequestView({report, parentReport, shouldShowHorizontalRule, trans
             </View>
 
             {hasReceipt && (
-                <View style={styles.moneyRequestViewImage}>
-                    <ReportActionItemImage
-                        thumbnail={receiptURIs.thumbnail}
-                        image={receiptURIs.image}
-                        enablePreviewModal
-                    />
-                </View>
+                <OfflineWithFeedback pendingAction={lodashGet(transaction, 'pendingAction')}>
```

Comment:
Maybe we can introduce...

```js
const pendingAction = lodashGet(transaction, 'pendingAction');

const getPendingFieldAction(fieldPath) => lodashGet(transaction, fieldPath) || pendingAction;
```

...as right now we're contributing to the duplication in this file. What do you think?